estate/__manifest__.py

```diff
@@ -2,8 +2,8 @@
     'name': 'Real Estate',
     'depends': ['base'],
     'data': ['security/ir.model.access.csv',
-             'views/estate_property_views.xml',
              'views/estate_tree.xml',
+             'views/estate_property_views.xml',
```

Comment:
Try to apply fixed in the commit that initially introduced the lines: once merge, readers will not be interrested in seeing the temporary states - they just want to see the actual solution that was found.
You can typically achieve this using `git rebase -i` and `git push --force-with-lease`.